site/src/scaffold/scripts/focusable-if-scrollable.ts

```diff
@@ -0,0 +1,32 @@
+export function handleFocusableIfScrollable(selectors = `[data-focusable-if-scrollable]`) {
+	const elements: HTMLElement[] = Array.from(document.querySelectorAll(selectors));
+
+	setScrollableAttrs(elements);
+
+	window.addEventListener('resize', () => {
+		const elements: HTMLElement[] = Array.from(
+			document.querySelectorAll('[data-focusable-if-scrollable]')
+		);
+		setScrollableAttrs(elements);
+	});
+}
+function setScrollableAttrs(elements: HTMLElement[]) {
+	for (const block of elements) {
+		const isHorizontallyScrollable = block.scrollWidth > block.clientWidth;
+		const isVerticallyScrollable = block.scrollHeight > block.clientHeight;
+
+		if (isHorizontallyScrollable) {
```

Comment:
Worth having a `isHorizontallyScrollable && isVerticallyScrollable` check that sets `aria-label` to `Scrollable element`?

Reply:
Yikes, don't know if we want to support it, since it is a bug in an of itself. Thoughts?

Reply:
I hear where you're coming from, but it seems odd to me that if there _is_ a case where a codeblock/similar element is scrollable both horizontally and vertically, then we're only communicating one of those directions.

That said, the likelihood of that happening is pretty infrequent, so this isn't a sticking point for me. I'd add the check, but if you feel it's not necessary, please feel free to resolve!